### PR TITLE
Add Our Journey So Far Section to the Homepage

### LINKED ISSUE #1150: Adding the Our journey so far Image from the  "New hire induction" Google slide
**Is your feature request related to a problem? Please describe.**
This is an enhancement for the "New content for the website" project 
https://sef.discourse.group/t/new-content-for-the-websites/362/8  

**Describe the solution you'd like**
Add the roadmap from the Google slide into the Home page Below the "our projects" section.

**Describe alternatives you've considered**
2bacf22d547cacb5c9e7dfc673f3a5558b0d755b

**Additional context**
Summarize the point form content into a paragraph (request assistance if needed)

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose
The purpose of this PR is to fix #1150   

## Goals
Adding the Our journey so far Image from the "New hire induction" Google slide.  

## Approach
Add the roadmap from the Google slide into the Home page Below the "our projects" section.

### Screenshots
Untitled  
  
### Preview Link
https://pr-1151-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by akeel230** [comment]
Great work @Madhawa97

**Comment by Madhawa97** [comment]
Thank you @akeel230

**Comment by Piumal1999** [comment]
The heading and the colours don't match with the design on the homepage. Is there any argon design component which we can use for this section?  @akeel230

**Comment by akeel230** [comment]
@Piumal1999 yah the data is intact but the design doesn't seem to fit in well. Thanks for pointing it out. 

@Madhawa97 how about we make a copy of the original google slide, then we make changes in that copy to match the theme of the website Home page. 
The data we have gathered won't change only the Heading and maybe we can use the fonts used in the slides to match the ones on the web Home page.

**Comment by Madhawa97** [comment]
@akeel230 yeah. That sounds great. I will do it and send you the link for that slide.

**Comment by Madhawa97** [comment]
by the way, I used `container-fluid` to fit the requested slide in the max-width possible (to be easier to read). Is there any comments on that? @akeel230 @Piumal1999

**Comment by Piumal1999** [comment]
> by the way, I used `container-fluid` to fit the requested slide in the max-width possible (to be easier to read). Is there any comments on that? @akeel230 @Piumal1999

Yeah. That's ok

**Comment by akeel230** [comment]
@Piumal1999 but the rest of the Home page has a border around ryt? @Madhawa97 are the letters unreadable if we add a border? I mean just the container class, not the fluid.

**Comment by akeel230** [comment]
and @Madhawa97 how is the mobile view when we add this section?

**Comment by Madhawa97** [comment]
Well, It's readable with some effort. but not easy.  
Web
either way, mobile view isn't readable without zooming in.
Mobile

**Comment by akeel230** [comment]
@Piumal1999  @Gravewalker666 @jayasanka-sack  what's ur thought? to optimize to mobiles obviously we can't use this as an image, right? 
Shall we just make the design suit the website design and merge the thing to show the information at least?
Also, what is the decision on the container vs container fluid class for this section?

**Comment by akeel230** [comment]
@Piumal1999 @Gravewalker666 @Madhawa97 @jayasanka-sack I made some changes in the slide 
Screenshot (743)

**Comment by Madhawa97** [comment]
Image updated with @akeel230's changes. also, the image is 1.6MB in size. If we are moving forward with this I think it's better to address that as well.

**Comment by Madhawa97** [comment]
Replaced the Our Journey So Far image with code. It now supports mobile view as well. would like to hear your suggestions. @akeel230 @Piumal1999

**Comment by akeel230** [comment]
@Madhawa97 superb stuff. @Piumal1999 I think this is great. the mobile view looks fantastic too.

File: index.html [hunk 1]
```diff
@@ -444,4 +512,32 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
         )
     })
 </script>
+<script>
+    (function () {
+        const timeline = document.querySelector(".timeline ol"),
```
Comment (jayasanka-sack) [inline index.html]: unused variable `timeline`

File: index.html [hunk 2]
```diff
@@ -444,4 +512,32 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
...
+            elH = document.querySelectorAll(".timeline li > div");
+
+        window.addEventListener("load", init);
+
+        function init() {
+            setEqualHeights(elH);
+        }
```
Comment (jayasanka-sack) [inline index.html]: Can be written as:
```suggestion
       $(function () {
           const elH = document.querySelectorAll(".timeline li > div");
           setEqualHeights(elH);
       }
```
btw, make variables meaningful. 
Also, you could move this into a separate js file dedicated to index.html. Internal js is discouraged.

File: index.html [hunk 3]
```diff
@@ -444,4 +512,32 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
...
+        
+        // SET EQUAL HEIGHTS
+        function setEqualHeights(el) {
+            let counter = 0;
+            for (let i = 0; i < el.length; i++) {
```
Comment (jayasanka-sack) [inline index.html]: can be replaced with `forEach()`

**Review by jayasanka-sack — COMMENTED** [review]
Good job @Madhawa97 !

I added some comments on the JS implementation.

Btw, the variable names are a bit ambiguous. Could you please rename them?

**Comment by Madhawa97** [comment]
Thank you @akeel230 and @jayasanka-sack . I'm working on it.

File: index.html (hunk 2 above)
Comment (Madhawa97) [reply index.html:524]: Roger that

File: index.html (hunk 3 above)
Comment (Madhawa97) [reply index.html]: Thank you

File: index.html (hunk 2 above)
Comment (Madhawa97) [reply index.html]: 👍🏻

**Comment by Madhawa97** [comment]
Pushed all the requested changes

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Madhawa97

**Comment by Madhawa97** [comment]
It's a pleasure 😇 @jayasanka-sack

**Pull request merged**